ONSdigital/census-rm-ddl
# Batch fulfilment print jobs

# Motivation and Context
Sending Fulfilment Print Jobs as one batch at 4pm daily so that they can be managed more easily

# What has changed
Added new table to action schema which will be populated throughout the day and then is emptied at 4pm (or whenever the cron occurs)

# How to test?
Run alongside other repos and check that the table is populated and then emptied

# Links
https://trello.com/c/W75HPE9U
